conda_smithy/lint_recipe.py

```diff
@@ -408,87 +410,18 @@ def run_conda_forge_specific(
     is_staged_recipes = recipe_dirname != "recipe"
 
     # 1: Check that the recipe does not exist in conda-forge or bioconda
-    if is_staged_recipes and recipe_name:
-        cf = gh.get_user(os.getenv("GH_ORG", "conda-forge"))
-
-        for name in set(
-            [
-                recipe_name,
-                recipe_name.replace("-", "_"),
-                recipe_name.replace("_", "-"),
-            ]
-        ):
-            try:
-                if cf.get_repo(f"{name}-feedstock"):
-                    existing_recipe_name = name
-                    feedstock_exists = True
-                    break
-                else:
-                    feedstock_exists = False
-            except github.UnknownObjectException:
-                feedstock_exists = False
-
-        if feedstock_exists and existing_recipe_name == recipe_name:
-            lints.append("Feedstock with the same name exists in conda-forge.")
-        elif feedstock_exists:
-            hints.append(
-                f"Feedstock with the name {existing_recipe_name} exists in conda-forge. Is it the same as this package ({recipe_name})?"
-            )
-
-        bio = gh.get_user("bioconda").get_repo("bioconda-recipes")
-        try:
-            bio.get_dir_contents(f"recipes/{recipe_name}")
-        except github.UnknownObjectException:
-            pass
-        else:
-            hints.append(
-                "Recipe with the same name exists in bioconda: "
-                "please discuss with @conda-forge/bioconda-recipes."
-            )
-
-        url = None
-        if recipe_version == 1:
-            for source_url in sources_section:
-                if source_url.startswith("https://pypi.io/packages/source/"):
-                    url = source_url
-        else:
-            for source_section in sources_section:
-                if str(source_section.get("url")).startswith(
-                    "https://pypi.io/packages/source/"
-                ):
-                    url = source_section["url"]
```

Comment:
@wolfv The offending code in staged-recipes was moved from this bit of smithy. I am guess @nichmor did the original work. Can he take a look and recommend a fix?